src/util/debug.c

```diff
@@ -287,13 +292,34 @@ void sss_vdebug_fn(const char *file,
          * searchable.
          */
         va_copy(ap_fallback, ap);
-        ret = journal_send(file, line, function, level, format, ap);
+        if (debug_chain_id > 0) {
+            result_fmt = chain_id_fmt_fixed;
+            ret = snprintf(chain_id_fmt_fixed, sizeof(chain_id_fmt_fixed),
+                           DEBUG_CHAIN_ID_FMT"%s", debug_chain_id, format);
+            if (ret < 0) {
+                return;
+            } else if (ret > sizeof(chain_id_fmt_fixed)) {
```

Comment:
man page: `a return value of size or more means that the output was truncated`
So I think it should be `>=`

Reply:
Fixed.